### PR TITLE
Issue #26 About us "Board of official" Section Completed

### LINKED ISSUE #25: About us "Who We Are" Section

### LINKED ISSUE #26: About us "Board of official" Section

### PR DESCRIPTION
**Pull request by shafdo**
Issue #26 About us "Board of official" Section Completed

Screenshot
image

TODO: The original board member data must be added to the `boardMembers.json` file.

### CODE REVIEW

**Comment by shafdo** [comment]
Issue #25 About us "Who We Are" Section Completed

image

File: src/components/client/about/boardSection/index.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-restricted-syntax */
```
Comment (MadhushaPrasad) [inline src/components/client/about/boardSection/index.jsx]: remove this eslint-disable options, please be kind to use eslint rules

File: src/components/client/about/boardSection/index.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+/* eslint-disable react/prop-types */
+/* eslint-disable react/function-component-definition */
+/* eslint-disable react/jsx-no-useless-fragment */
+
+import { useState } from 'react'
+import boardMembers from './boardMembers.json'
+import facebook from '../../../../assets/images/AboutPage/facebook.svg'
+import instagram from '../../../../assets/images/AboutPage/instagram.svg'
```
Comment (MadhushaPrasad) [inline src/components/client/about/boardSection/index.jsx]: use fontawesome for these social media icons

File: src/components/client/about/boardSection/index.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      <div className="row">
+        <div className="col">
+          <div className="d-block my-5">
+            <h1 className="board-heading">Board of officials - 2022</h1>
+          </div>
+
+          <div className="d-flex justify-content-center my-5">
+            <select name="" id="board-year" onChange={ArrangeBoard}>
+              <option value="2023">2023</option>
+              <option value="2022">2022</option>
```
Comment (MadhushaPrasad) [inline src/components/client/about/boardSection/index.jsx]: set default value as 2022 in here

File: src/components/client/about/boardSection/index.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    <div id="board-section" className="container">
+      <img className="board-patton-blue" src={pattonBlue} alt="" />
+      <div className="row">
+        <div className="col">
+          <div className="d-block my-5">
+            <h1 className="board-heading">Board of officials - 2022</h1>
+          </div>
+
+          <div className="d-flex justify-content-center my-5">
+            <select name="" id="board-year" onChange={ArrangeBoard}>
```
Comment (MadhushaPrasad) [inline src/components/client/about/boardSection/index.jsx]: remove border in the select element, style it like the Figma design

File: src/components/client/about/boardSection/index.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+const SocialInstagram = (props) => {
+  const { profileLink } = props
+  return (
+    <a href={profileLink} target="_blank" className="social-link" rel="noreferrer">
+      <img className="social-icon" src={instagram} alt="instagram" />
+    </a>
+  )
+}
+
+const SocialLinkedin = (props) => {
```
Comment (MadhushaPrasad) [inline src/components/client/about/boardSection/index.jsx]: remove these functions and replace those with fontawesome icons

File: src/components/client/about/whoWeAreSection/index.jsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable react/jsx-no-useless-fragment */
```
Comment (MadhushaPrasad) [inline src/components/client/about/whoWeAreSection/index.jsx]: remove eslint disable options

File: src/components/client/common/header/index.jsx [hunk 7]
```diff
@@ -46,7 +46,7 @@ export default function Header() {
                 </Link>
               </li>
               <li className="nav-item">
-                <a href="src/components/client/header#" className="nav-link">
+                <a href="/about" className="nav-link">
```
Comment (MadhushaPrasad) [inline src/components/client/common/header/index.jsx]: replace anchor tag with Link., use react-router-dom for navigations

File: src/pages/AboutPage.jsx [hunk 8]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/jsx-no-useless-fragment */
```
Comment (MadhushaPrasad) [inline src/pages/AboutPage.jsx]: remove these also

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]
- [ ] use social media icons by fontawesome, dont use as svg icons
- [ ] fix left side padding as in our figma file
image

- [ ] remove border in 'select year' dropdown element -> make it as figma file -> you can get that styles from figma inspect section
- [ ] bind board of official data to all of cards remove this images.
image

- [ ] remove eslint-disable and use our eslint rules
- [ ] fix conflicting files

**Pull request closed without merging**